src/main/java/software/amazon/smithy/lsp/SmithyInterface.java

```diff
@@ -46,29 +46,30 @@ public static Either<Exception, ValidatedResult<Model>> readModel(Collection<Fil
       Model.Builder builder = Model.builder();
 
       URL[] urlArray = externalJars.stream().map(SmithyInterface::fileToUrl).toArray(URL[]::new);
+      URLClassLoader urlClassLoader = new URLClassLoader(urlArray);
 
-      if (urlArray.length > 0) {
```

Comment:
I can revert this particular chunk, as the only thing I really need outside of it is the `urlClassLoader` value, but I think the condition is redundant: if the array is empty, we'll just add "no shapes".